docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -1,36 +1,131 @@
-# Using global profile configuration with Zowe Explorer
+# Zowe Explorer Profile Configuration
 
-## Feature overview
+Zowe Explorer @Next enables you to use the team configuration file that centralizes and simplifies your profile management.
 
-The Zowe Explorer vNext release enables you to adopt the Team Configuration File (recently developed by the [Zowe CLI Squad](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md)) - designed to make profile management more centralized and much simpler.
+> @Next is developed by the [Zowe CLI Squad](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md).
+
+You can configure global profiles and project-specific profiles.
 
 ## Prerequisites
 
-- Install [Zowe CLI @next version](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#installing-next-version).
-- [Initialize](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#initializing-global-configuration) the Global Configuration file. The resulting `zowe.config.json` and `zowe.schema.json` files will be placed in your Zowe home directory.
-- [Customize](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#editing-configuration) the Global Configuration file.
+Meet the following software requirements before you use the team configuration file:
+
+1. Install [Zowe CLI @next version](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#installing-next-version).
```

Comment:
@crawr Can we add the list of supported CLI version here? Maybe a side by side list of the ZE version and its CLI version counterpart

Reply:
| Zowe Explorer Version    | Zowe SDK Version        |
|--------------------------|-------------------------|
| v2.0.0-next.202107151328 | 7.0.0-next.202106012053 |
| TBD                      | 7.0.0-next.202108202027 |